src/common/config/config.service.ts

```diff
@@ -169,6 +169,10 @@ export class ConfigService {
     return this.config.get(`lto.${this.networkName(network)}.node`);
   }
 
+  getNetworkId(): string {
+    return this.config.get('lto.networkId');
+  }
+
```

Comment:
Relay works for both testnet and mainnet. It determines the network based on the address. You don't need to configure a specific network for the service.

If for some reason, you do want to limit the use of the service to only mainnet or testnet, you'll need to modify this all through the app.